sgkit/tests/io/vcf/utils.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+from sgkit.typing import PathType
+
+
+def path_for_test(shared_datadir: Path, file: str, is_path: bool = True) -> PathType:
+    """Return a test data path whose type is determined by `is_path`.
+
+    If `isPath` is True, return a `Path`, otherwise return a `str`.
+    """
+    path: PathType = shared_datadir / file
```

Comment:
A more idiomatic way to say this would be:

```
path: PathType = shared_datadir / file
return path if is_path else str(path)
```

Reply:
I will make the changes to functionality in a separate PR , so this one is solely about migrating the existing code.